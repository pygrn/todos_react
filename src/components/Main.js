import React from 'react'

const Main = () => (
  <div className="container">
    <div className="row">
      <div className="col s12">
        <h1>Pyton Girona</h1>
      </div>
    </div>
  </div>
)
export default Main