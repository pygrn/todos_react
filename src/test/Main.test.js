import React from 'react'
import Main from '../components/Main'
import renderer from 'react-test-renderer'

test('Main', () => {
    const component = renderer.create(<Main/>)
    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})