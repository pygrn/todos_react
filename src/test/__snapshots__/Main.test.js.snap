// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main 1`] = `
<div
  className="container"
>
  <div
    className="row"
  >
    <div
      className="col s12"
    >
      <h1>
        Python Girona
      </h1>
    </div>
  </div>
</div>
`;
